lib/statement.rb

```diff
@@ -0,0 +1,28 @@
+class Statement
+
+  attr_reader :transactions_list
+
+  def initialize(transactions_list)
+    @transactions_list = transactions_list
+  end
+
+  def generate_statement
+    create_header
+    display_records(transactions_list)
+  end
+
+  private
+
+  def create_header
+    puts "date | debit | credit | balance"
+  end
+
+  def display_records(arr)
```

Comment:
It feels like this method has two responsibilities.  Have a think about what they are and if you could separate them.